src/css/_modebar.scss

```diff
@@ -5,12 +5,12 @@
     z-index: 1001;
 }
 
-.modebar--hover {
+.modebar--hover > :not(.watermark) {
```

Comment:
Apparently `:not` is supported IE9+: https://caniuse.com/#search=%3Anot

Reply:
CSS child selector should also work in IE9+. The reason we use it is because we cannot override the `opacity` of a child if the parent has a CSS `opacity`.

Reply:
Nice find. But we should test this out in IE9 and IE11 (from browserstack) before merging this thing.

Reply:
Works in IE11:
![ie11_watermark](https://user-images.githubusercontent.com/301546/48923987-753e2900-ee81-11e8-88cf-594b4fe6b9e1.png)

IE9/10 doesn't work on master :neutral_face: 
![ie9_fail_master2](https://user-images.githubusercontent.com/301546/48923999-8edf7080-ee81-11e8-886c-707dd020e2c5.png)

I will open an issue for the latter.

Reply:
Oh wait, right the main plotly.js bundle doesn't work in ie9.

You'll need to use `plotly-basic.js`:

- https://cdn.plot.ly/plotly-basic-latest.min.js, or
- from https://github.com/plotly/plotly.js/blob/master/lib/index-basic.js

Reply:
Works in IE9 :tada: 

![ie9_watermark](https://user-images.githubusercontent.com/301546/48954982-c0a41600-ef19-11e8-8021-973d502a4e11.png)

About using `plotly-basic.js` in IE9 versus `plotly.js`: is this documented somewhere? If not I should open an issue over at the documentation project.

Reply:
More precisely, all [SVG-based traces](https://github.com/plotly/plotly.js/blob/922bad611cd8a48f77ab2086d6a935b9eebab6f5/test/jasmine/bundle_tests/ie9_test.js#L3-L15) should work in IE9. 

We could maybe add some info here ->

https://github.com/plotly/plotly.js/tree/master/dist#to-support-ie9